dask/dataframe/io/parquet.py

```diff
@@ -121,7 +117,8 @@ def read_parquet(path, columns=None, filters=None, categories=None, index=None,
                         columns=[c for c in pf.columns if c in dtypes])
 
     for cat in categories:
```

Comment:
Not one that doesn't exist, I mean one that isn't a categorical. It's important to ensure that the metadata created matches exactly - no extra/missing columns, and dtypes are all correct.